db/seeds.rb

```diff
@@ -21,3 +21,17 @@
   small_cover_url: '/tmp/futurama.jpg', categories: [family, comedy, sci_fi]})
 monk = Video.create({title: 'Monk',description: 'A detective series about a neurotic detective', 
   small_cover_url: '/tmp/monk.jpg', large_cover_url: 'tmp/monk_large.jpg', categories: [drama, crime]})
+south_park = Video.create({title: 'South Park',description: 'A rude comedy series', 
```

Comment:
I would prefer to format this code like:

```ruby
south_park = Video.create!(
  title: 'South Park',
  description: 'A rude comedy series',
  small_cover_url: '/tmp/south_park.jpg',
  categories: [comedy]
)
```
Its more readable and also `create!` throws an exception when the record creation fails in seeds. Right now, you are not informed about it.